app/src/androidTest/java/com/google/jetpackcamera/VideoRecordingDeviceTest.kt

```diff
@@ -104,6 +155,23 @@ internal class VideoRecordingDeviceTest {
         )
     }
 
+    private fun deleteFilesInDirAfterTimestamp(timeStamp: Long): Boolean {
+        var hasDeletedFile = false
+        for (file in File(ImageCaptureDeviceTest.DIR_PATH).listFiles()) {
```

Comment:
Shouldn't this use a DIR_PATH that uses `getExternalStoragePublicDirectory(Environment.DIRECTORY_MOVIES)`?

Reply:
No longer valid